simd/CMakeLists.txt

```diff
@@ -30,9 +30,6 @@ if(CPU_TYPE STREQUAL "x86_64")
   if(CYGWIN)
     set(CMAKE_ASM_NASM_OBJECT_FORMAT win64)
   endif()
-  if(CMAKE_C_COMPILER_ABI MATCHES "ELF X32")
-    set(CMAKE_ASM_NASM_OBJECT_FORMAT elfx32)
-  endif()
```

Comment:
Why did you remove this?  It has nothing to do with MIPS.  You just broke the build on a completely unrelated platform.

Reply:
Sorry. That was a copy and paste issue.